src/main/scala/thurloe/database/ThurloeDatabaseConnector.scala

```diff
@@ -163,11 +173,13 @@ case object ThurloeDatabaseConnector extends DataAccess {
     }
   }
 
-  def set(userKeyValuePair: UserKeyValuePair): Future[DatabaseOperation] = {
-    Aes256Cbc.encrypt(userKeyValuePair.keyValuePair.value.getBytes("UTF-8"), secretKey) match {
-      case Success(encryptedValue) => databaseWrite(userKeyValuePair, encryptedValue)
-      case Failure(x) => Future.failed(x)
-    }
+  def set(userKeyValuePairs: UserKeyValuePairs): Future[DatabaseOperation] = {
+    Future(Await.result(database.run(databaseWrites(userKeyValuePairs).map { operations =>
```

Comment:
this feels crappy but I don't know if it was worth refactoring all of the database code to use `inTransaction`.

should I just do it or file a ticket to do it later?